src/test/java/SalaryCalculatorTestMockito.java

```diff
@@ -0,0 +1,34 @@
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import java.io.IOException;
+
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class SalaryCalculatorTestMockito {
```

Comment:
This test has no sense. You didn't get it correctly what for Mockito is.
It is for mocking class' dependency, not for mocking exactly class that need to be tested.